nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -106,11 +107,16 @@ controller_server:
       required_movement_radius: 0.5
       movement_time_allowance: 10.0
     # Goal checker parameters
-    goal_checker:
+    #precise_goal_checker:
+    #  plugin: "nav2_controller::SimpleGoalChecker"
+    #  xy_goal_tolerance: 0.25
+    #  yaw_goal_tolerance: 0.25
+    #  stateful: True
+    general_goal_checker:
       plugin: "nav2_controller::SimpleGoalChecker"
-      xy_goal_tolerance: 0.25
-      yaw_goal_tolerance: 0.25
-      stateful: True
+      xy_goal_tolerance: 0.35
+      yaw_goal_tolerance: 0.35
+      stateful: False
```

Comment:
Looking at the diff, the "general" one was stateful true and the tolerances were both `0.25`, can we change back to that?

Reply:
This should be already done.